media/js/newsletter/form-protocol.js

```diff
@@ -0,0 +1,163 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    // TODO port this JS to Protocol
```

Comment:
This is pretty much ported directly from our [basket client example](https://github.com/mozilla/basket-example/), but I had to change a few things as some of the newsletter classes and IDs have changed since it became a Protocol component. Could use some review / testing.

Reply:
Hmm, I'll dig into this and see if i can figure out what's up. nice catch!

Reply:
I think this issue is that you only get a confimration email if you've never signed up before using a given email address. I sucessfully get the email using my mozilla email with a random modifier e.g. myname+baskettest@mozilla.com, so I think everything is working ok.

That said, it would still be great if @pmac could cast an eye here, to make sure I haven't missed any critical parameters.

Reply:
I'm taking a look, but your "same email" theory was my first thought too Alex. I tend to avoid using my real email addresses for testing these days (even with a `+` modifier) and go with `<anything-i-want>@restmail.net`. [Restmail](http://restmail.net/) is your friend, and basket will automatically mark the entries created using that domain as testing data. Just FYI :)

Reply:
I used a throwaway alias @spamgourmet.com address, I wonder if basket/salesforce ignores that because it has the word "spam" in it?

Reply:
I don't know of anything that would look for that, but this is salesforce... so ¯\_(ツ)_/¯

Reply:
Fwiw, I tried https://www.guerrillamail.com/ and got the same issue (no reply)